packages/website/src/components/Playground.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useReducer, useState } from 'react';
 import type Monaco from 'monaco-editor';
 import clsx from 'clsx';
-import useThemeContext from '@theme/hooks/useThemeContext';
+import { useColorMode } from '@docusaurus/theme-common';
```

Comment:
@Josh-Cena am I doing this right?

I recall you mentioning that we could probably un-swizzle some components? If there's a big refactor I'd be down to take that as a followup.

Reply:
Hmmm... Did I? :D

And yes, you are doing it right.

Reply:
haha awesome, thanks!